backend/src/data/migrations/20220818224217_add_fullname_to_user_details.ts

```diff
@@ -0,0 +1,38 @@
+import { Knex } from 'knex';
+
+enum ColumnName {
+  FULL_NAME = 'full_name',
+  FIRST_NAME = 'first_name',
+  LAST_NAME = 'last_name',
+}
+
+enum TableName {
+  USERS = 'users',
+  USER_DETAILS = 'user_details',
+}
+
+async function up(knex: Knex): Promise<void> {
+  await knex.schema.table(TableName.USER_DETAILS, (table) => {
+    table.string(ColumnName.FULL_NAME).notNullable();
+    table.dropColumn(ColumnName.FIRST_NAME);
+    table.dropColumn(ColumnName.LAST_NAME);
+  });
+
+  await knex.schema.table(TableName.USERS, (table) => {
+    table.dropColumn(ColumnName.FULL_NAME);
+  });
+}
```

Comment:
@PavelSukhinin so I need to copy all existing fullNames from user table to userDetails, correct? could you please give me a hint on how can I do it?

Reply:
As for now we have empty `user_details` table so it could be something like
```
WITH user_details_to_insert AS (
  SELECT id AS "user_id", full_name
  FROM users
)
INSERT INTO user_details
  SELECT * FROM user_details_to_insert;
```
And on rollback you could move `full_name` col values back to `users` table by something like
```
UPDATE users SET full_name = (
  SELECT full_name FROM user_details WHERE users.id = user_details.user_id
)
```

Reply:
@andanfe 👍 
@kalinkaaaa14 we can
1. create full_name column in user_details
2. update user_details table from users table
3. drop full_name in users table, drop first and last name columns in user_details

Reply:
thanks to all